fedimint-client/src/module/mod.rs

```diff
@@ -143,6 +143,42 @@ where
         self.module_instance_id
     }
 
+    /// Turn a typed output into a dyn version
+    pub fn make_dyn_output(&self, output: <M::Common as ModuleCommon>::Output) -> DynOutput {
+        output.into_dyn(self.module_instance_id())
+    }
+
+    /// Turn a typed input into a dyn version
+    pub fn make_dyn_input(&self, input: <M::Common as ModuleCommon>::Input) -> DynInput {
+        input.into_dyn(self.module_instance_id())
+    }
```

Comment:
nit: why not impl using `make_dyn`? This could reduce refactoring effort should we ever change its inner workings.